### PR TITLE
multiarch: Move multiarch building to bpfman-operator repo

### PR DESCRIPTION
**Pull request by Billy99**
Since bpfman-operator is moving to it's own repo, no longer build and push the bpfman-operator and bpfman-agent from the bpfman repo. Also remove any bpfman-operator specific checks.

Note: The Kubernetes Integration Test were being run out of the bpfman-operator directory, so all those tests are dropped (they are still being run out of hte bpfman-oprator repo against bpfman:latest).

### CODE REVIEW

**Review by anfredette — COMMENTED** [review]
LGTM, but I'll let @astoycos approve it.

**Review by astoycos — APPROVED** [review]
Looks good on this side, holding until we merge the other one

**Pull request merged**
